### ISSUE TITLE
no release, cant find nfldb.sql

### ISSUE THREAD

**Issue by chppppp** [derek-adair/nfldb#13]
i am looking for a copy of the db but i cant find it in the repo. thanks!

**Comment by AccidentalGuru** [comment]
I found it when I git-clone'd it, but couldn;t find it navigating the repo.

**Comment by hardyaf** [comment]
When I git cloned it, it still wasn't there, unless I am having a complete brain fart.. Any chance you could upload a 2019 version? Doesn't really matter to me if the game from this Thursday is present. It would be greatly appreciated.

**Comment by hardyaf** [comment]
> I found it when I git-clone'd it, but couldn;t find it navigating the repo.

When I git cloned it, it still wasn't there, unless I am having a complete brain fart.. Any chance you could upload a 2019 version? Doesn't really matter to me if the game from this Thursday is present. It would be greatly appreciated.

**Comment by omarcoming** [comment]
You have to unzip nfldb2019.sql.zip or you can redownload it here: https://github.com/derek-adair/nfldb/releases/download/1.0.0a4/nfldb2019.sql.zip

But I am having trouble setting up the database

**Comment by RSABryan** [comment]
Can anyone help with a sample command line to pg_restore this file?  I'm trying different combinations but figured someone on here has probably already done this...  Many thanks.

**Comment by AccidentalGuru** [comment]
Last I got it working I used Ye Olde nfldb installation instructions not the new fancy Docker one b/c I'm older than fancy and had limited HD space. You can find all the details here:

https://github.com/BurntSushi/nfldb/wiki/Installation

To answer your exact question. . .
If you have the nfldb.sql.zip, you can unzip as usual for your sys (e.g. Mac/Lin/Win). 
Then cd to the data directory you use to start postgresql and where you have unzippewd nfldb.sql and run:

`psql -U nfldb nfldb < nfldb.sql`

oh, and I have only once gotten it to work with an encrypted password, and I thunk that was back in the python 2.7 days. 2.7 wasn't pretty, but man did it run just about anything.

**Comment by alex-heim** [comment]
I have been able to restore the database from the sql file posted, but it doesn't seem to populate the games table. When I attempt to connect through python I get the following error:

AssertionError: Schema has version 0 but is not empty.

Has anyone else found a way to fix this?
